lms/static/styles/components/_LMSGrader.scss

```diff
@@ -1,39 +1,35 @@
 .LMSGrader {
   display: flex;
   flex-direction: column;
-  font-size: 18px;
+  font-size: 16px;
   height: 100%;
   width: 100%;
   position: fixed;
-  color: white;
-  .LMSGrader__course-assignment {
-    h2, h3 {
-      margin-left: 20px;
-    }
-  }
+  color: $grey-6;
 }
 
 .LMSGrader__grading-components {
-  background-color: $grey-7;
-  padding: 0;
+  background-color: white;
```

Comment:
Do we have a variable for "white"? Can we use transparent if not?

Reply:
turns out we do, but I'm not sure why? just makes us type an extra char and I can't see a reason why we would ever have $white enver be white. hmm... But I like transparent better in any case

Reply:
I think I can just remove it, but I also don't understand the use case of $white since it should always just be white. Seems like extra stuff